Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -1497,10 +1497,7 @@ - (void)URLSession:(NSURLSession *)session webSocketTask:(NSURLSessionWebSocketT
         cookieStorage = storageSession->nsCookieStorage();
 
 #if HAVE(CFNETWORK_OVERRIDE_SESSION_COOKIE_ACCEPT_POLICY)
-    // We still need to check the selector since CFNetwork updates and WebKit updates are separate
-    // on older macOS.
```

Comment:
As you are touching this code anyway, please remove `CFNETWORK_OVERRIDE_SESSION_COOKIE_ACCEPT_POLICY` everywhere.